go.mod

```diff
@@ -129,15 +126,17 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+// when adding/remove replace, remove the following block of indirect dependencies
+// and run `go mod tidy -compat=1.20` (based on go version above)
+require cloud.google.com/go v0.110.2 // indirect
```

Comment:
why moving it outside the first required section?

Reply:
so weird. no idea how that happened. thanks for catching that.

Reply:
I cleaned it up :-)